pkg/api/config_test.go

```diff
@@ -75,7 +75,7 @@ func TestValidateTests(t *testing.T) {
 			id: "test without `commands`",
 			tests: []TestStepConfiguration{
 				{
-					As: "test",
+					As:                         "test",
```

Comment:
oops, you have go 1.11? The tests will fail. Please use `1.10`

Reply:
eek, we need to update

Reply:
This one is painful. Need to reconfigure my env to be dual-go-lang-version :////